roles/network-multus/templates/sriovdp.yml

```diff
@@ -27,6 +46,39 @@ spec:
         operator: Exists
         effect: NoSchedule
       serviceAccountName: sriov-device-plugin
+      initContainers:
+      - name: config-container
+        image: fedora:29
+        resources:
+          requests:
+            cpu: "100m"
+            memory: "50Mi"
+          limits:
+            cpu: "100m"
+            memory: "50Mi"
+        command:
+        - /bin/bash
+        - -c
+        args:
+        - |
+          if [ -f /etc/sriov-nodes-config/$NODE_NAME ]; then
+            config_path=/etc/sriov-nodes-config/$NODE_NAME
+          else
+            config_path=/etc/sriov-nodes-config/fallback-config
+          fi
+          cp $config_path /etc/pcidp/config.json
+        volumeMounts:
+        - name: config
+          mountPath: /etc/pcidp/config.json
+        - name: configmap
+          mountPath: /etc/sriov-nodes-config/config.json
```

Comment:
I think you need to change this to `/etc/sriov-nodes-config/` then it will add the fallback-config in the right place